lms/static/sass/views/_login-register.scss

```diff
@@ -111,51 +97,35 @@ $sm-btn-linkedin: #0077b5;
 
             .text {
```

Comment:
I would prefer that this special .text positioning be associated with the &.lines rule above since it relates to the line in the background of the title. Can you move it inside there? (looks like line 82)